src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --maincolor: #00acdc;
+  --header-height: 60px;
+  --header-margin: 50px;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .logo {
+    display: flex;
+    align-items: center;
+    margin-left: var(--header-margin);
+  }
+
+  .nav {
+    margin: 0 var(--header-margin) 0 0;
+  }
+
+  .nav ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    padding: 0;
+
+    :not(:last-child) {
+      margin-right: 20px;
+    }
```

Comment:
The `:not(:last-child)` pseudo-class should not be nested inside `.nav ul`. As with the previous comments, nesting is not a feature of standard CSS and should be corrected in the final stylesheet.

Reply:
nesting IS a feature NOW